blackbox/docs/src/doc_tests/process_test.py

```diff
@@ -275,6 +275,13 @@ def test_graceful_stop_none(self):
         self.set_settings({"cluster.graceful_stop.min_availability": "none"})
         decommission(self.clients[0], self.node_names[0])
 
+        # make sure that node is down
+        time.sleep(15)
```

Comment:
I don't particularly like this, any recomendations?

Reply:
Shouldn't the retry below wait for the node to be gone ?